src/lib/CRSF/CRSF.h

```diff
@@ -487,7 +487,7 @@ class CRSF
     void ICACHE_RAM_ATTR setSentSwitch(uint8_t index, uint8_t value);
 
 ///// Variables for OpenTX Syncing //////////////////////////
-#define OpenTXsyncPakcetInterval 100 // in ms
+#define OpenTXsyncPakcetInterval 250 // in ms
```

Comment:
Is this actually in ms...? Why 250ms?

Reply:
typo - can we fix the name while we're making changes? (Pakcet -> Packet)

Reply:
Felt like 10hz was abit too spammy
lets settle for 5hz, bench tested it and it didn't make a difference